.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+jobs:
+  build:
+    test-py2:
+      docker:
+        - image: circleci/python:2.7.17
+      steps:
+        - checkout
+        - run: make test
+    test-py3:
+      docker:
+        - image: circleci/python:3.7.6
+      steps:
+        - checkout
+        - run: make test
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - test-py2
+      - test-py3
```

Comment:
Does this need a new line at the EOF?

Reply:
Also good call testing both py2 and py3 for backwards compatibility!

Reply:
Added for cleanliness -- and yeah hopefully saves us from slipping on one or the other going forward.